descriptors/ModuleDescriptor-template.json

```diff
@@ -727,7 +747,10 @@
         "roles.item.delete",
         "policies.item.post",
         "policies.item.put",
-        "policies.item.delete"
+        "policies.item.delete",
+        "role-capabilities.collection.post",
+        "role-capabilities.collection.put",
+        "role-capabilities.collection.delete"
```

Comment:
also system user permissions should be updated with new endpoinds permissions both for policies and roles with capabilities and capability sets. 
It should be updated both in ModuleDescriptor metadata and in system-user-permissions until we found a way to have them in the single place